indico/modules/rb_new/client/js/common/bookings/BookingDetailsModal.jsx

```diff
@@ -200,7 +200,8 @@ class BookingDetailsModal extends React.Component {
         const legendLabels = [
             {label: Translate.string('Occurrence'), color: 'orange'},
         ];
-        const showActionButtons = (!isCancelled && !isRejected);
+        const showAccept = canAccept && !isAccepted;
```

Comment:
I think that should be covered by `canAccept`. but we can revisit the logic here when cleaning up the permission checks in the backend and/or actually adding functionality to those buttons